core/primitives-core/src/version.rs

```diff
@@ -173,6 +174,7 @@ impl ProtocolFeature {
             ProtocolFeature::RejectBlocksWithOutdatedProtocolVersions => 132,
             #[cfg(feature = "protocol_feature_simple_nightshade_v2")]
             ProtocolFeature::SimpleNightshadeV2 => 135,
+            ProtocolFeature::PostStateRoot => 136,
```

Comment:
Note that the nightly protocol version is `138`.

I guess it is fine to re-use protocol version numbers of stabilized features.